app/forms/personal_details_form.rb

```diff
@@ -41,7 +41,7 @@ def date_of_birth
   end
 
   def save!
-    if valid?
+    if valid?(:save)
```

Comment:
Not sure about adding the `:save` context. Does it change the behaviour? The specs all pass if I remove it. The context argument is usually `:create`, `:update` or a custom context.

Reply:
this is a remnant, it can go in the bin